### ISSUE TITLE
[Bug] [a11y] Consider making the "Common myths" item from PB larger

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#7708]
Prerequisites

Make sure you have Google Accessibility Scanner installed.
Steps to reproduce

    Go to private mode and make sure no tabs are opened.
    Using the Accessibility Scanner scan the Private Browsing Home screen.

Expected behavior

No suggestions are made.
Actual behavior

There is a suggestion to consider increasing the "Common myths about private browsing" clickable item - as seen in the screenshot below.
Device information

    Android device: Samsung Galaxy Tab S3 (Android 8)
    Fenix version: RC 3.0.2, Beta build 3.1.0-beta 3, Nightly build from 01/15

Screenshot_20200115-082612_Accessibility Scanner

![image](https://user-images.githubusercontent.com/4400286/73442926-fe8a9e80-4355-11ea-81ae-f90a6774019b.png)

**Comment by sblatz** [comment]
Make this say "button" (it should be a button or link object)
It should have 48dp.
